WinRT.Runtime/MatchingRefApiCompatBaseline.net5.0.txt

```diff
@@ -1,3 +1,6 @@
 Compat issues with assembly WinRT.Runtime:
 TypesMustExist : Type 'System.Numerics.VectorExtensions' does not exist in the reference but it does exist in the implementation.
-Total Issues: 1
+MembersMustExist : Member 'public System.Boolean WinRT.IObjectReference.Resurrect()' does not exist in the reference but it does exist in the implementation.

```

Comment:
These will require an assembly version bump, but I wonder if we can avoid it by making the APIs internal?  Any reason they should be public?

Reply:
good call - done